Source/Core/DolphinQt/Config/Graphics/AdvancedWidget.cpp

```diff
@@ -145,72 +145,67 @@ void AdvancedWidget::OnEmulationStateChanged(bool running)
+  static const char TR_DUMP_TEXTURE_DESCRIPTION[] = QT_TR_NOOP(
+      "Dumps decoded game textures to User/Dump/Textures/<game_id>/.\n\nIf unsure, leave "
+      "this unchecked.");
   static const char TR_LOAD_CUSTOM_TEXTURE_DESCRIPTION[] = QT_TR_NOOP(
-      "Load custom textures from User/Load/Textures/<game_id>/.\n\nIf unsure, leave this "
+      "Loads custom textures from User/Load/Textures/<game_id>/.\n\nIf unsure, leave this "
       "unchecked.");
   static const char TR_CACHE_CUSTOM_TEXTURE_DESCRIPTION[] =
-      QT_TR_NOOP("Cache custom textures to system RAM on startup.\nThis can require exponentially "
+      QT_TR_NOOP("Caches custom textures to system RAM on startup.\nThis can require exponentially "
                  "more RAM but fixes possible stuttering.\n\nIf unsure, leave this unchecked.");
-  static const char TR_DUMP_EFB_DESCRIPTION[] =
-      QT_TR_NOOP("Dump the contents of EFB copies to User/Dump/Textures/.\n\nIf unsure, leave this "
-                 "unchecked.");
+  static const char TR_DUMP_EFB_DESCRIPTION[] = QT_TR_NOOP(
+      "Dumps the contents of EFB copies to User/Dump/Textures/.\n\nIf unsure, leave this "
+      "unchecked.");
   static const char TR_DISABLE_VRAM_COPIES_DESCRIPTION[] =
       QT_TR_NOOP("Disables the VRAM copy of the EFB, forcing a round-trip to RAM. Inhibits all "
                  "upscaling.\n\nIf unsure, leave this unchecked.");
   static const char TR_INTERNAL_RESOLUTION_FRAME_DUMPING_DESCRIPTION[] = QT_TR_NOOP(
-      "Create frame dumps and screenshots at the internal resolution of the renderer, rather than "
+      "Creates frame dumps and screenshots at the internal resolution of the renderer, rather than "
       "the size of the window it is displayed within. If the aspect ratio is widescreen, the "
-      "output "
-      "image will be scaled horizontally to preserve the vertical resolution.\n\nIf unsure, leave "
-      "this unchecked.");
+      "output image will be scaled horizontally to preserve the vertical resolution.\n\nIf "
+      "unsure, leave this unchecked.");
 #if defined(HAVE_FFMPEG)
   static const char TR_USE_FFV1_DESCRIPTION[] =
-      QT_TR_NOOP("Encode frame dumps using the FFV1 codec.\n\nIf unsure, leave this unchecked.");
+      QT_TR_NOOP("Encodes frame dumps using the FFV1 codec.\n\nIf unsure, leave this unchecked.");
 #endif
   static const char TR_FREE_LOOK_DESCRIPTION[] = QT_TR_NOOP(
-      "This feature allows you to change the game's camera.\nMove the mouse while holding the "
-      "right "
-      "mouse button to pan and while holding the middle button to move.\nHold SHIFT and press "
-      "one of "
-      "the WASD keys to move the camera by a certain step distance (SHIFT+2 to move faster and "
-      "SHIFT+1 to move slower). Press SHIFT+R to reset the camera and SHIFT+F to reset the "
-      "speed.\n\nIf unsure, leave this unchecked.");
+      "Allows manipulation of the in-game camera. Move the mouse and hold the right button "
```

Comment:
I'm missing the "while"s in this. "move and hold" is a little ambiguous, imo.

Reply:
Hm? You marked this as resolved, and thumbs uped it, without re-adding whiles? I'm confused, what did you adjust  here? Do you disagree?

Reply:
Oh, I'm sorry! I agree, and I fixed it locally, but I held off on pushing any commits until you had answered the question about the rendering description. I guess that's confusing for other people, I'll do it differently in the future.

Sorry for the confusion!